documentation/specs/TabNav.md

```diff
@@ -0,0 +1,81 @@
+# `TabNav` Component Specification
+
+## Overview
+
+A `TabNav` is a set of styled links that let users navigate between related pages.
+
+---
+
+## Design
```

Comment:
note that the horizontal overflow behavior of this component (the responsive design in Figma) is still in flux. due to WAI ARIA reflow guidelines, that behavior may change to render overflowed items into a disclosure menu instead of using scroll. however the overflow is handled shouldn't cause a change in the API design of the component.